lib/rucio/client/replicaclient.py

```diff
@@ -41,46 +41,43 @@ def quarantine_replicas(self, replicas, rse=None, rse_id=None):
         if (rse is None) == (rse_id is None):
             raise ValueError("Either RSE name or RSE id must be specified, but not both")
 
-        data = {'rse': rse, 'rse_id': rse_id, 'replicas': replicas}
         url = build_url(self.host, path='/'.join([self.REPLICAS_BASEURL, 'quarantine']))
         headers = {}
-        r = self._send_request(url, headers=headers, type_='POST', data=dumps(data))
-        if r.status_code // 100 != 2:
-            exc_cls, exc_msg = self._get_exception(headers=r.headers, status_code=r.status_code, data=r.content)
-            raise exc_cls(exc_msg)
+        chunk_size = 1000
+        for i in range(0, len(replicas), chunk_size):
+            chunk = replicas[i:i + chunk_size]
+            data = {'rse': rse, 'rse_id': rse_id, 'replicas': chunk}
+            r = self._send_request(url, headers=headers, type_='POST', data=dumps(data))
+            if r.status_code // 100 != 2:
+                exc_cls, exc_msg = self._get_exception(headers=r.headers, status_code=r.status_code, data=r.content)
+                raise exc_cls(exc_msg)
```

Comment:
That's right, chunking has nothing to do with adding the flag, but I thought I would do it to avoid inter-dependencies between patches. So I would like to keep the chunking here.